scr/example-01b_simple-beam-3-span_point-load.py

```diff
@@ -0,0 +1,303 @@
+# PyNite Example
+# Example: Simply supported beam with a point load
+# Units: inches and kips, use consistent units
+
+from operator import ge
+import numpy as np
+from PyNite import FEModel3D
+from PyNite import Reporting
+from PyNite import Visualization
+from pathlib import Path
+import matplotlib.pyplot as plt
+from itertools import accumulate
+
+# Updated parameters for this plot
+plt.style.use("../plt/figstylefile.mplstyle")
+
+
+class Section:
+    def __init__(
```

Comment:
This line is formatted different from the rest of PyNite. That's fine, but I'd rather pull it into a separate branch than "master" so that I can format it to match before pulling it into the master branch.